webpack/assets/stylesheets/application/partials/_navigation.scss

```diff
@@ -63,7 +62,7 @@
 }
 
 .navbar-pf-vertical {
-  background: image-url('navbar.png');
+  background: url('#{$local-img-path}/navbar.png');
```

Comment:
this will break the theme-ability? /cc @ShimShtein ?

Reply:
The only thing that I can think of, is the changes to folder structure. The theme should reflect it for sure. Basically the same rule will apply for the new path too  - as long as the file name is the same, it will be taken from the theme.

By the way, if webpack generates file hashes like rails - it should generate the same hash as for the main foreman.